pkg/kubelet/config/file.go

```diff
@@ -173,32 +173,32 @@ func (s *sourceFile) extractFromDir(name string) ([]*v1.Pod, error) {
 	for _, path := range dirents {
 		statInfo, err := os.Stat(path)
 		if err != nil {
-			klog.Errorf("Can't get metadata for %q: %v", path, err)
+			klog.ErrorS(err, "Could not get metadata", "path", path)
 			continue
 		}
 
 		switch {
 		case statInfo.Mode().IsDir():
-			klog.Errorf("Not recursing into manifest path %q", path)
+			klog.ErrorS(nil, "Not recursing into manifest path", "path", path)
```

Comment:
Not sure we should be using ErrorS here as there isn't an error.

Reply:
we shouldn't change log level based on [this comment](https://github.com/kubernetes/kubernetes/issues/98976#issuecomment-791734797)